packages/terra-clinical-result/tests/wdio/clinical-result-spec.js

```diff
@@ -0,0 +1,214 @@
+// Terra.describeViewports('Clinical Result', ['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], () => {
+/*
+Terra.describeViewports('Clinical Result', ['medium'], () => {
```

Comment:
Can we separate each component into their own spec so it's clearer what is being tested?